models/unit_tests.go

```diff
@@ -72,6 +73,18 @@ func createTestEngine(fixturesDir string) error {
 	return InitFixtures(&testfixtures.SQLite{}, fixturesDir)
 }
 
+func forcedRemoveAll(dir string) error {
```

Comment:
Could you rename this method? It's not really forcing anything

Reply:
Any suggestions?

Reply:
Maybe something like this?:

```
func retryingRemoveAll(dir string, retries int) error {
```

Reply:
@Bwko done